universal-login-wallet/src/ui/react/App.tsx

```diff
@@ -22,6 +23,8 @@ const App = () => {
       <Route exact path="/terms" component={TermsAndConditionsScreen} />
       <Route exact path="/privacy" component={PrivacyPolicy} />
       <Route exact path="/create" component={CreateFlow} />
+      <Route exact path="/connectionSucceed" component={ConnectionSuccess} />
```

Comment:
We have a component named `ConnectionSuccess` and we are creating url named `connectionSucceed`. It hurts...  I see 2 directions: 
- Rename this path to `connectinSuccess` and then make PR with rename
- PR with rename ConnectionSuccess -> ConnectionSucceed and after that, we can merge this PR (1617)

Reply:
✔️